node/Gruntfile.js

```diff
@@ -4,7 +4,7 @@ module.exports = function(grunt) {
   grunt.initConfig({
     nodeunit: {
       tests: [
-        'test/unit/*.js'
```

Comment:
I will copy the "nodeunit:only" task from t2-cli, then we can avoid changes like this